src/main/java/com/jcertif/pic/sonar/oauth/OAuthPlugin.java

```diff
@@ -35,6 +35,8 @@
 import org.sonar.api.ServerExtension;
 import org.sonar.api.SonarPlugin;
 import org.sonar.plugins.oauth.api.OAuthClient;
+import org.sonar.colorizer.CaseInsensitiveKeywordsTokenizer;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'org.sonar.colorizer.CaseInsensitiveKeywordsTokenizer'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AUselessImportCheck)